### DISCUSSION TITLE
[Support]: Counting cars in a static image during set interval

### DISCUSSION THREAD

**Discussion by danodemano** [blakeblackshear/frigate#11876] [General Support]
### Describe the problem you are having

I hope this is the right place to ask and I assume the answer is no but I'm in search of a solution and since Frigate is already up and running with a Coral TPU I'd like to leverage that if possible. 

My state has public camera feeds for various state routes and freeways. One particular interchange I pass through a lot is under construction and traffic fluctuates a lot. I would love to have this as a "car count" sensor in Home Assistant so I can determine if I go that way or not. I already have the Frigate/MQTT/HA integration working.

Hopefully that makes sense. Here is a link to the camera feed I'm hoping to use. I'd like to pull that image every 15-30 seconds and count the cars then pass that to HA. https://itscameras.dot.state.oh.us/images/CMH/I-71_at_US-36.jpg

Thanks so much!

### Version

0.13.2-6476F8A

### Frigate config file

```yaml
N/A
```

### Relevant log output

```shell
N/A
```

### FFprobe output from your camera

```shell
N/A
```

### Operating system

Other Linux

### Install method

Docker CLI

### Coral version

USB

### Network connection

Wired

### Camera make and model

N/A

**Comment by hawkeye217** [comment]
The Frigate HA integration provides a `sensor.<camera_name>_<object_name>_count` sensor for each camera and object that shows the number of objects of a type currently in the frame. That sensor should get you what you need.

**Reply by danodemano** [reply]
Thanks for that. I guess the rest of the puzzle though how would I go about grabbing that image every say 30 seconds and processing it for vehicles? Can a static image even be used as a camera?

**Reply by NickM-27** [reply]
is there a reason why more frequent updates would be a problem?

**Reply by danodemano** [reply]
More frequent updates wouldn't be a problem at all, just wasn't sure if that would add too much load to anything. Currently with 12 cameras:

Inference Speed: 27.96 ms
CPU: 55.5%
Memory: 0.2%
